packages/preview-middleware-client/test/__mock__/sap/m/Text.ts

```diff
@@ -0,0 +1,7 @@
+export default class Text{
+    text = 'sample-text'
```

Comment:
Is this text used somewhere, why do you need it here?

Reply:
resolved in [e94a899](https://github.com/SAP/open-ux-tools/pull/1477/commits/e94a89919304cadb80bf98ae1eb4ece1069cc8cb)